src/endpoints/sources.js

```diff
@@ -270,6 +270,24 @@ const putNarrative = contentTypesConsumed([
 ]);
 
 
+/* Group customizations
+ */
+
+/* GET
+ */
+const getGroupOverview = contentTypesProvided([
+  ["text/markdown", sendGroupOverview()],
+  ["text/plain", sendGroupOverview()],
+]);
+
+
+/* PUT
+ */
+const putGroupOverview = contentTypesConsumed([
+  ["text/markdown", receiveGroupOverview()],
+]);
```

Comment:
I'm also curious why this "worked" before 8aa733ca29f41f5eab8b2edbf8a060d966dbd937, when using `contentTypesProvided` instead of `contentTypesConsumed`.

Reply:
Oh duh, this "worked" before 9168b758615c8163486b6c8a6cee2e9415116b32.